twilio/modules/listener/Module.md

```diff
@@ -149,140 +41,61 @@ Callback URL registration method depends on the event type.
-configurable string & readonly twilioAuthToken = ?;
-configurable string & readonly fromNumber = ?;
-configurable string & readonly toNumber = ?;
-configurable string & readonly test_message = ?;
-configurable string & readonly twimlUrl = ?;
-configurable string & readonly callbackUrl = ?;
-configurable int & readonly port = ?;
+    configurable string & readonly twilioAuthToken = ?;
+    configurable string & readonly callbackUrl = ?;
+    configurable int & readonly port = ?;
 
-listener twilioListener:Listener tListener = new (port, twilioAuthToken, callbackUrl);
-
-service / on tListener {
-    remote function onSmsQueued(twilioListener:SmsStatusChangeEvent event) returns error? {
-        log:printInfo("Queued", event);
-    }
-    remote function onSmsSent(twilioListener:SmsStatusChangeEvent event) returns error? {
-        log:printInfo("Sent", event);
-    }
-    remote function onSmsDelivered(twilioListener:SmsStatusChangeEvent event) returns error? {
-        log:printInfo("Delivered", event);
+    listener twilioListener:Listener tListener = new (port, twilioAuthToken, callbackUrl);
+```
+### Step 3 - Add a service 
+1. Add a service with necessary remote functions as below
+```ballerina
+    service / on tListener {
+        remote function onSmsDelivered(twilioListener:SmsStatusChangeEvent event) returns error? {
+            log:printInfo("Delivered", event);
+        }
     }
-}
+```
+2. Use `bal run` command to compile and run the Ballerina program.
 
-public function main() {
-    twilio:TwilioConfiguration twilioConfig = {
-        accountSId: twilioAccountSid,
-        authToken: twilioAuthToken
-    };
-    twilio:Client twilioClient = new (twilioConfig);
-    var details = twilioClient->sendSms(fromNumber, toNumber, test_message, callbackUrl);
-    if (details is error) {
-        log:printInfo(details.message());
+## Quick reference
+Code snippets of some frequently used functions: 
+* QUEUED/SENT/DELIVERED SMS Event
+This examples shows how you can start a ballerina twilio listener using localhost. you will need to use ngork to expose a web server running on your local machine to the internet.
+```ballerina
+    service / on tListener {
+        remote function onCallRang(twilioListener:CallStatusChangeEvent event) returns error? {
+            log:printInfo("Ringing", event);
+        }
+        remote function onCallAnswered(twilioListener:CallStatusChangeEvent event) returns error? {
+            log:printInfo("In Progress", event);
+        }
+        remote function onCallCompleted(twilioListener:CallStatusChangeEvent event) returns error? {
+            log:printInfo("Completed", event);
+        }
     }
 }
 ```
-### Ringing/InProgress/Completed Voice Call Events
+* Ringing/InProgress/Completed Voice Call Events
 This provides the listener support to the voice calls. you will need to use ngork to expose a web server running on your local machine to the internet. Find more sample from here.
```

Comment:
Suggestion: "This provides the listener support to the voice calls. You need to use ngork to expose a web server running on your local machine to the internet."

Reply:
Is "Find more sample from here." a duplication of the link added after all the code snippets?